Core/include/Acts/EventData/TrackHelpers.hpp

```diff
@@ -0,0 +1,53 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#pragma once
+
+#include "Acts/EventData/MultiTrajectory.hpp"
+#include "Acts/EventData/TrackContainer.hpp"
+
+namespace Acts {
+
+/// Helper function to calculate a number of track level quantities and store
+/// them on the track itself
+/// @tparam track_container_t the track container backend
+/// @tparam track_state_container_t the track state container backend
+/// @tparam holder_t the holder type for the track container backends
+/// @param track A track proxy to operate on
+template <typename track_container_t, typename track_state_container_t,
+          template <typename> class holder_t>
+void calculateTrackQuantities(
```

Comment:
@pbutti

Reply:
LGTM. Only comment would be to add in the description some words about the "false" in the template arguments. What templated argument is that?

Reply:
They're unfortunately necessary to accept *any* kind of track container, not just a concrete one. I tried to clarify this with the comment above this:

```cpp
/// @tparam track_container_t the track container backend
/// @tparam track_state_container_t the track state container backend
/// @tparam holder_t the holder type for the track container backends
/// @param track A track proxy to operate on
```

Reply:
You have four template parameters here:
```cpp
Acts::TrackProxy<track_container_t, track_state_container_t, holder_t, false>
```
You explain what the first three are, but you don't explain why the final one is false. This is, I assume, what @pbutti is referring to?

I guess this is duplicated documentation from `TrackProxy` but it probably wouldn't hurt to say that since it will be modified, the passed `TrackProxy` must be writable, i.e. have `read_only` = false?

Reply:
I see, and yes that's correct. I can add a note to the documentation of the `@param` (because it's not a template parameters to this functions).

Reply:
Yes, I meant add it to `@param track`